tutorials/models/2_small_graph/3_tree-lstm.py

```diff
@@ -370,7 +372,6 @@ def batcher_dev(batch):
             epoch, step, loss.item(), acc))
 
 ##############################################################################
-# To train the model on full dataset with different settings(CPU/GPU,
-# etc.), please refer to our repo's
-# `example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
-# Besides, we also provide an implementation of the Child-Sum Tree LSTM.
+# To train the model on a full dataset with different settings (such as CPU or GPU),
+# please refer to the `PyTorch example <https://github.com/dmlc/dgl/tree/master/examples/pytorch/tree_lstm>`__.
+# Besides, you also provide an implementation of the Child-Sum Tree-LSTM.

```

Comment:
it's DGL-team that provide an implementation of the Child-Sum Tree-LSTM instead of user.

Reply:
@john-andrilla Would you mind changing the last sentence to "Besides, there is also an implementation of the Child-Sum Tree-LSTM." ? Thanks.

Reply:
Understood. addA